lib/config/definitions.js

```diff
@@ -1624,7 +1624,11 @@ const options = [
     stage: 'package',
     type: 'object',
     default: {
-      fileMatch: ['\\.gradle$', '(^|/)gradle.properties$'],
+      fileMatch: [
+        '\\.gradle(\\.kts)?$',
+        '(^|/)gradle.properties$',
+        '(^|/)buildSrc/.*',
```

Comment:
Why is this needed in the `fileMatch`?

Reply:
As @jcornaz said in #3054, it is becoming popular that defining dependencies/versions in a Kotlin source files under `buildSrc`. For example:

- https://handstandsam.com/2018/02/11/kotlin-buildsrc-for-better-gradle-dependency-management/
- https://proandroiddev.com/gradle-dependency-management-with-kotlin-94eed4df9a28

Even though I don't handle contents such as:

```kotlin
object Versions {
    val support_lib = "27.0.2"
    val retrofit = "2.3.0"
    val rxjava = "2.1.9"
}

object Libs {
    val support_annotations = "com.android.support:support-annotations:${Versions.support_lib}"
    val support_appcompat_v7 = "com.android.support:appcompat-v7:${Versions.support_lib}"
    val retrofit = "com.squareup.retrofit2:retrofit:${Versions.retrofit}"
    val retrofit_rxjava_adapter = "com.squareup.retrofit2:adapter-rxjava2:${Versions.retrofit}"
    val rxjava = "io.reactivex.rxjava2:rxjava:${Versions.rxjava}"
}
``` 

but this PR should work against more simple cases such as:

```kotlin
object Libs {
    private val supportLibVersion = "27.0.2"
    private val retrofitVersion = "2.3.0"
    private val rxjavaVersion = "2.1.9"

    val support_annotations = "com.android.support:support-annotations:${supportLibVersion}"
    val support_appcompat_v7 = "com.android.support:appcompat-v7:${supportLibVersion}"
    val retrofit = "com.squareup.retrofit2:retrofit:${retrofitVersion}"
    val retrofit_rxjava_adapter = "com.squareup.retrofit2:adapter-rxjava2:${retrofitVersion}"
    val rxjava = "io.reactivex.rxjava2:rxjava:${rxjavaVersion}"
}
```

Reply:
I reconsidered this. As you said in https://github.com/renovatebot/renovate/issues/3054#issuecomment-452850852, not adding `buildSrc` to the `fileMatch` and letting users do that by themselves will be a good (at least for now) as a first step.

Reply:
I've just omitted the entry.

Reply:
@rarkins I've just omitted the entry.